pkg/database/database.go

```diff
@@ -14,19 +14,25 @@ import (
 
 	"github.com/Azure/ARO-RP/pkg/api"
 	"github.com/Azure/ARO-RP/pkg/database/cosmosdb"
+	"github.com/Azure/ARO-RP/pkg/encrypt"
 	"github.com/Azure/ARO-RP/pkg/env"
 )
 
+// encryptionSecretName must match key name in the service keyvault
+const encryptionSecretName = "db-encryption-key"
```

Comment:
maybe better in pkg/env

Reply:
Either here Or in the env and we should have `GetDatabaseKey()`. Up to you :)